### PR TITLE
Mirgrate to Slidev nearform template

### LINKED ISSUE #288: Replace nf-present with Slidev
At the moment, the Fastify-workshop uses nf-present to build the slide-deck for presentation.

A Nearform [slidev ](https://sli.dev/)template is available [here](https://github.com/nearform/new-promises-workshop) which has better support for content formatting and already has NF design.

Migrate the Fastify workshop from using nf-present to slidev

### PR DESCRIPTION
**Pull request by sameer-coder**
This PR changes the presentation format to use Nearform theme. nf-present presentation library is replaced with sli.dev library because it offers much customization options and the content fits better.

Closes #288

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
   "license": "CC-BY-SA-4.0",
   "author": "Simone Busoli <simone.busoli@nearform.com>",
   "scripts": {
-    "start": "nf-present deck.md --css=./styles.css --out=index.html --watch",
```
Comment (simoneb) [inline package.json:11]: remove nf-present

**Comment by sameer-coder** [comment]
The CI is failing because of some vulnerabilities in slidev-cli. I think we can ignore them.

File: package.json [hunk 2]
```diff
@@ -8,17 +8,21 @@
...
     "db:down": "docker-compose down",
     "lint": "eslint --ext .ts,.js ./src",
     "test": "TS_NODE_FILES=1 tap --ts"
   },
   "dependencies": {
-    "@nearform/nf-present": "^2.0.0",
     "@nearform/sql": "^1.9.0",
     "@sinclair/typebox": "^0.23.3",
+    "@slidev/cli": "^0.28.5",
+    "@slidev/theme-default": "^0.21.1",
```
[the remark this answers is not in the document]
Comment (sameer-coder) [reply package.json]: Nope. I just removed it

**Comment by simoneb** [comment]
@RafaelGSS can you give this a go?

File: slides.md [hunk 3]
```diff
@@ -1,106 +1,134 @@
...
 
----
+</div>
 
-class: branded
+---
 
 # Ecosystem
 
+<div class="flex-row">
```
Comment (RafaelGSS) [inline slides.md:89]: All the slides need this class? Wouldn't be better to adjust it automatically in the theme?

Comment (sameer-coder) [reply slides.md]: It turns out that even if I don't use the class the slides look the same. Seems like this is the default class 🙂

File: slides.md [hunk 4]
```diff
@@ -1,106 +1,134 @@
...
+© Copyright 2019-2022 NearForm Ltd. All Rights Reserved.
 
-class: branded
+</div>
+
+---
 
 # Introduction: Why Fastify
 
+<div>
```
Comment (RafaelGSS) [inline slides.md:24]: Maybe the `div` is not necessary as well, when slidev parse to HTML, the `div` is automatically added.

Comment (sameer-coder) [reply slides.md]: Good point. I have removed all the divs

**Pull request merged**